tests/src/system/basic/WskBasicTests.scala

```diff
@@ -348,6 +348,14 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.rule, ruleName) {
                 (rule, name) =>
                     rule.create(name, trigger = triggerName, action = actionName)
+
+                    // to validate that the rule was created enabled, we do an update and expect CONFLICT
+                    rule.create(name, trigger = triggerName, action = actionName, update = true, expectedExitCode = CONFLICT)
```

Comment:
i don't think we can use expectedExitCode = ANY_EXIT_CODE, because the rule creation logic needs to know whether it should look for "ok" on stdout. either we need to eliminate the stdout check, or we need to (somehow) information rule.create of the expected failure